app/views/my/page.html.erb

```diff
@@ -28,7 +28,7 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 <% breadcrumb_paths(l(:label_my_page)) %>
 <% content_for :action_menu_specific do %>
-  <%= link_to l(:label_personalize_page), action: 'page_layout', accesskey: accesskey(:edit) %>
+  <%= link_to l(:label_personalize_page), { :action => 'page_layout', :accesskey => accesskey(:edit) }, "accesskey" => accesskey(:edit) %>
```

Comment:
Why did you replace the new hash-syntax with the old one?

new: `foo: 'bar'`
old: `:foo => 'bar'`